recipes/http-classes-with-new/src/workflow.ts

```diff
@@ -0,0 +1,72 @@
+import { getNodeImportStatements } from '@nodejs/codemod-utils/ast-grep/import-statement';
+import { getNodeRequireCalls } from '@nodejs/codemod-utils/ast-grep/require-call';
+import { resolveBindingPath } from '@nodejs/codemod-utils/ast-grep/resolve-binding-path';
+import type { SgRoot, Edit, SgNode } from '@codemod.com/jssg-types/main';
+
+const classNames = [
```

Comment:
```suggestion
const CLASS_NAMES = [
```

It's a constant so let's notice it on casing also maybe add JSdoc